src/Spectre.Algorithms/Results/DivikResult.cs

```diff
@@ -216,18 +231,24 @@ private DivikResult[] ParseSubregions(MWStructArray array)
             }
             return null;
         }
+
         #endregion
 
         #region Save
+
         /// <summary>
         /// Saves result under the specified path in JSON format.
         /// </summary>
         /// <param name="path">The destination path.</param>
-        /// <exception cref="System.NotImplementedException">Always, as result saving has not been implemented yet.</exception>
         public void Save(string path)
         {
-            throw new NotImplementedException("Result saving has not been implemented yet.");
+            string data = JsonConvert.SerializeObject(this);
+            using (StreamWriter sw = File.CreateText(path))
```

Comment:
Nevertheless, The WriteAllText is just a wrapper around the StreamWriter so it just introduces additional overhead. Do we need such change?